optax/_src/alias_test.py

```diff
@@ -586,7 +586,9 @@ def test_plain_preconditioning(self):
     expected_precond_vec = precond_mat.dot(
         vec, precision=jax.lax.Precision.HIGHEST
     )
-    chex.assert_trees_all_close(plain_precond_vec, expected_precond_vec)
+    chex.assert_trees_all_close(
+      plain_precond_vec, expected_precond_vec, rtol=1e-5
```

Comment:
Use four spaces when wrapping a call of a function that spans beyond the line limit (see PEP 8), otherwise the internal linting flags this and we need to patch it. So just use
```
    chex.assert_trees_all_close(
        plain_precond_vec, expected_precond_vec, rtol=1e-5
    )
    
```

Reply:
@vroulet Fixed.